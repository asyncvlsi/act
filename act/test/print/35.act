defproc test()
{
    pint W = 65;

    chp {
        log(1 << (65 - 1));
        log(1 << (W - 1))
    }
}

test t;
