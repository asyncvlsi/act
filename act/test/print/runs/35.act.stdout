defproc test ();

defproc test ()
{

/* instances */

/* connections */
chp {
log(0x0000000000000000000000000000000010000000000000000);log(0x0000000000000000000000000000000010000000000000000)
}
}


/* instances */
test t;

/* connections */
